core/config/project_settings.cpp

```diff
@@ -701,6 +701,11 @@ Error ProjectSettings::_setup(const String &p_path, const String &p_main_pack, b
 		}
 	}
 
+#ifdef ANDROID_ENABLED
+	// Attempt to load sparse PCK assets.
+	_load_resource_pack("res://assets.sparsepck", false, 0, true);
+#endif
+
```

Comment:
Can you update the comment at the top of this function and reference this additional logic for Android

Reply:
Done.